### PR TITLE
add deploy predict test

### PR DESCRIPTION
**Pull request by Ash-Lee233**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: deploy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from .predict import detect
+
+__all__ = ['detect']
```
Comment (zhanghuiyao) [inline deploy/__init__.py:3]: 末尾空行

File: tests/modules/test_deploy_predict.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+           'fire hydrant', 'stop sign', 'parking meter', 'bench', 'bird', 'cat', 'dog', 'horse', 'sheep', 'cow',
+           'elephant', 'bear', 'zebra', 'giraffe', 'backpack', 'umbrella', 'handbag', 'tie', 'suitcase', 'frisbee',
+           'skis', 'snowboard', 'sports ball', 'kite', 'baseball bat', 'baseball glove', 'skateboard', 'surfboard',
+           'tennis racket', 'bottle', 'wine glass', 'cup', 'fork', 'knife', 'spoon', 'bowl', 'banana', 'apple',
+           'sandwich', 'orange', 'broccoli', 'carrot', 'hot dog', 'pizza', 'donut', 'cake', 'chair', 'couch',
+           'potted plant', 'bed', 'dining table', 'toilet', 'tv', 'laptop', 'mouse', 'remote', 'keyboard', 'cell phone',
+           'microwave', 'oven', 'toaster', 'sink', 'refrigerator', 'book', 'clock', 'vase', 'scissors', 'teddy bear',
+           'hair drier', 'toothbrush' ]
+    draw_result(image_path, result_dict, names, save_path=save_path)
+
```
Comment (zhanghuiyao) [inline tests/modules/test_deploy_predict.py:48]: 测试用例最后可以加个判断(iou & class is), 
assert iou > 0.5
assert class_id == 'id of cat'

Comment (Ash-Lee233) [reply tests/modules/test_deploy_predict.py:49]: iou在detect的方法里有设定阈值0.65
我加了一个cat的assert

Comment (zhanghuiyao) [reply tests/modules/test_deploy_predict.py:49]: detect 里面的 iou_thres 是用来做nms用的，最后可以对 预测值 和 真实值 做个iou判断

**Review by zhanghuiyao — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**